src/Controller/ModulesController.php

```diff
@@ -96,7 +96,7 @@ public function index(): ?Response
 
         $this->ProjectConfiguration->read();
 
-        $objects = (array)$response['data'];
+        $objects = $this->Modules->objects((array)$response);
```

Comment:
We could introduce an `ApiFormatter` component (via WebTools) to embed included resources/objects in `relationships` section of the response and avoid this method. 
@didoda more details via call later

Reply:
in the direction of your proposal: https://github.com/bedita/web-tools/pull/38

when that PR will be merged, we could update this PR as necessary to use the `ApiFormatter`.